DigitalLearningSolutions.Data/Services/RegistrationService.cs

```diff
@@ -258,16 +244,18 @@ public void PromoteDelegateToAdmin(AdminRoles adminRoles, int? categoryId, int d
                 delegateUser.ProfileImage
             );
 
-            // TODO HEEDLS-900 these user IDs are placeholders and should be updated
-            registrationDataService.RegisterAdmin(adminRegistrationModel, 0);
+            var userId = userDataService.GetUserIdFromDelegateId(delegateId);
+            registrationDataService.RegisterAdmin(adminRegistrationModel, userId);
         }
 
         public string CreateAccountAndReturnCandidateNumber(DelegateRegistrationModel delegateRegistrationModel)
         {
             try
             {
                 ValidateRegistrationEmail(delegateRegistrationModel);
-                return registrationDataService.RegisterNewUserAndDelegateAccount(delegateRegistrationModel);
+                var candidateNumber =
+                    registrationDataService.RegisterNewUserAndDelegateAccount(delegateRegistrationModel);
+                return candidateNumber;

```

Comment:
was this change necessary?

Reply:
Oops that was for debugging purposes, removed